packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -196,17 +196,49 @@ module.exports.external = [
         throw "#if requires exactly one argument";
       }
 
+      const allValuesAreNull = (obj) => {
```

Comment:
we werent handling two cases in our ifs:
1. objects with a single null value
2. objects with a series of nested objects all with null values